packages/pn-commons/src/components/Notifications/StatusTooltip.tsx

```diff
@@ -26,6 +26,7 @@ const StatusTooltip = ({
         id={`status-chip-${label}`}
         label={label}
         color={color}
+        role='cell'
```

Comment:
This is not completely correct. The problem is into the CustomTooltip component. There we do this

```
{cloneElement(children, {
            onClick: handleTooltipOpen,
          })}
```
In this way we are cloning the children component and we are setting the onClick props. In our case the children component is the Chip component and a Chip component with onClick props behaves as a button (https://mui.com/material-ui/react-chip/#chip-actions).
To avoid this we have to avoid the onClick props if openOnClick props is undefined.